config/packages/doctrine.yaml

```diff
@@ -1,22 +1,15 @@
-parameters:
-    # Adds a fallback DATABASE_URL if the env var is not set.
-    # This allows you to run cache:warmup even if your
-    # environment variables are not available yet.
-    # You should not need to change this value.
-    env(DATABASE_URL): ''
-
 doctrine:
     dbal:
-        # configure these for your database server
-        driver: 'pdo_mysql'
-        server_version: '5.7'
+        url: '%env(resolve:DATABASE_URL)%'
+
+        # IMPORTANT: You MUST configure your server version,
```

Comment:
@danrot think you need to add `?serverVersion=5.7` to the appveyor and travis configuration.

Travis run [postgresql](https://docs.travis-ci.com/user/database-setup/#postgresql) on `9.2`